internal/kafka/test/common/kafka_pollers.go

```diff
@@ -61,6 +64,7 @@ func WaitForKafkaCreateToBeAccepted(ctx context.Context, db *db.ConnectionFactor
 			}
 		}).
 		OnRetry(func(attempt int, maxRetries int) (done bool, err error) {
+			//nolint: bodyclose
```

Comment:
Is there a reason why we are ignoring lint here?

Reply:
Thanks. It'll be good to document the `WaitForKafkaCreateToBeAccepted` method as such. Current it does not say anything about this.